recipes-support/collision-avoidance-library/coav-control_0.0.1.bb

```diff
@@ -0,0 +1,41 @@
+# coav-control.bb
+SUMMARY = "Collision avoidance test application."
+HOMEPAGE = "https://github.com/01org/collision-avoidance-library"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
+
+DEPENDS = "glm librealsense python-future libcoav"
+RDEPENDS_${PN} = "librealsense libcoav"
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "gitsm://git@github.com/01org/collision-avoidance-library.git;protocol=https;branch=master"
+
+S = "${WORKDIR}/git"
+
+inherit cmake pythonnative
+
+EXTRA_OECMAKE = "-DWITH_TOOLS=ON -DWITH_REALSENSE=ON"
+
+do_configure_prepend() {
+    export PYTHONPATH="${PKG_CONFIG_SYSROOT_DIR}/usr/lib/python2.7/site-packages/"
+}
+
+do_install () {
+    install -d ${D}${bindir}
+    install -m 0755 ${B}/tools/coav-control/coav-control ${D}${bindir}
+    chrpath -d ${D}${bindir}/coav-control
```

Comment:
Oh... Just now I noticed this is do_install() and not do_install_append(). I have a quote somewhere to add here.... Wait a minute .